.github/workflows/check.yml

```diff
@@ -18,6 +18,21 @@ jobs:
           mdl -s .github/linter/readme_style.rb README.md
           mdl -s .github/linter/base_style.rb .github
 
+  cairo-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Rust nightly
+        uses: dtolnay/rust-toolchain@nightly
+        with:
+          components: rustfmt, clippy
+      - name: Install cairo-lint
+        run: cargo install scarb-cairo-lint --git https://github.com/keep-starknet-strange/cairo-lint
+
+      - uses: asdf-vm/actions/install@v3
+      - run: scarb cairo-lint
+      - run: scarb cairo-lint --fix
```

Comment:
Is this run needed? The changes wouldn't persist to the branch right?

Reply:
I was also thinking to remove the `run: scarb cairo-lint --fix` but wanted to here your feedback on this